src/Compilers/Core/Portable/CommandLine/CommonCommandLineArguments.cs

```diff
@@ -21,12 +21,12 @@ namespace Microsoft.CodeAnalysis
     /// </summary>
     public abstract class CommandLineArguments
     {
-        internal bool IsScriptRunner { get; set; }
+        public bool IsScriptRunner { get; protected internal set; }
```

Comment:
:bulb: This property should be documented since it's being moved to the public API surface for the first time.